src/mixins/wallet.js

```diff
@@ -96,5 +108,37 @@ export default {
         'menubar=no,location=no,width=576,height=768'
       );
     },
+    exportFollowerList() {
+      const header = [
+        this.$t('portfolio_follower_export_ID'),
+        this.$t('portfolio_follower_export_wallet'),
+      ];
+      const contents = this.populatedFollowers.map(
+        ({ displayName, wallet }) => [this.escapeCsvField(displayName), wallet]
+      );
+
+      // Convert list to CSV string
+      const csvString = `${header.join(',')}\n${contents
+        .map(row => row.join(','))
+        .join('\n')}`;
+
+      const csvBlob = new Blob([csvString], {
+        type: 'text/csv;charset=utf-8;',
+      });
+      // Download CSV file
+      const csvUrl = URL.createObjectURL(csvBlob);
+      const hiddenLink = document.createElement('a');
+      hiddenLink.href = csvUrl;
+      hiddenLink.target = '_blank';
+      hiddenLink.download = 'my-followers.csv';
+      hiddenLink.click();
+    },
+
+    escapeCsvField(field) {
```

Comment:
Should move to util?
```suggestion
    escapeCSVField(field) {
```